### ISSUE TITLE
So... what can we do exactly?

### ISSUE THREAD

**Issue by eksynn** [Tommsy64/SmartMovingReloaded#15]
just wondering what can be done, currently, and how to do it
the original mod had a key you could use to activate specific movements, configurable through the in-game menu. we don't really have an in-game menu in this version, so i was wondering how to wall-climb, crouch, etc

also, any word on that discord server?

**Comment by Xorbah** [comment]
If you'd have read the description, the only thing implemented thus far is crawling.

**Comment by ScienceDrummer** [comment]
Yeah, that's about it. The camera doesn't shift when you crawl however, so it doesn't look like you're crawling but you actually are.

**Comment by torzod** [comment]
> Yeah, that's about it. The camera doesn't shift when you crawl however, so it doesn't look like you're crawling but you actually are.

Trying out this mod currently, and the camera seems to shift, but with no animation, making it hard to tell when you are crawling vs sneaking because the change in viewing height is so drastic

**Comment by FelipeCardona95** [comment]
The mod doesnt needs Render PLayerAPi? Im tyring the 0.0.4 alpha and I dont see it requires anything also in the controls I dont see any bind to crawl
